Jenkinsfile

```diff
@@ -660,6 +660,10 @@ pipeline {
                 // Previous preview deployment needs to be scaled to 0 to be replaced correctly
                 sh "kubectl -n ${PREVIEW_NAMESPACE} scale deployment nuxeo-preview --replicas=0"
               }
+              sh """kubectl create secret generic kubernetes-docker-cfg \
+                  --namespace=${PREVIEW_NAMESPACE} \
+                  --from-literal=.dockerconfigjson="\$(kubectl --namespace platform get secret kubernetes-docker-cfg -ojsonpath='{.data.\\.dockerconfigjson}' | base64 --decode)" \
+                  --type=kubernetes.io/dockerconfigjson --dry-run -o yaml | kubectl apply -f -"""
```

Comment:
why not simply
```
kubectl create secret generic kubernetes-docker-cfg \
                  --namespace=${PREVIEW_NAMESPACE} \
                  --from-literal=.dockerconfigjson="\$(kubectl --namespace platform get secret kubernetes-docker-cfg -ojsonpath='{.data.\\.dockerconfigjson}' | base64 --decode)" \
                  --type=kubernetes.io/dockerconfigjson
```

Reply:
Not sure it works if I paste some bash code directly in a script directive which takes groovy initially?
I can also try and remove it but I wanted to stay consistent with the all the other kubectl calls accros the same file which are wrapped in an sh""" """" call. 
wdyt ?

Reply:
@ataillefer is just simplifying the content of the `sh"""`, not saying you should remove the `sh"""` part :)